Makefile

```diff
@@ -137,20 +108,13 @@ test-docker:
 	@echo ">> testing docker image"
 	./test_image.sh "$(DOCKER_IMAGE_NAME):$(DOCKER_IMAGE_TAG)" 9100
 
-$(GOPATH)/bin/promtool promtool:
+$(FIRST_GOPATH)/bin/promtool promtool:
 	@GOOS= GOARCH= $(GO) get -u github.com/prometheus/prometheus/cmd/promtool
 
-$(GOPATH)/bin/promu promu:
-	@GOOS= GOARCH= $(GO) get -u github.com/prometheus/promu
-
-$(GOPATH)/bin/staticcheck:
-	@GOOS= GOARCH= $(GO) get -u honnef.co/go/tools/cmd/staticcheck
-
-
 .PHONY: all style format build test test-e2e vet tarball docker promtool promu staticcheck checkmetrics
```

Comment:
This `.PHONY` also needs to be updated to remove the stuff from `Makefile.common`.

Reply:
fixed